src/java.base/share/classes/sun/nio/ch/Util.java

```diff
@@ -43,7 +43,7 @@
 
 public class Util {
 
-    private static final JavaNioAccess NIO_ACCESS = SharedSecrets.getJavaNioAccess();
+    static final JavaNioAccess NIO_ACCESS = SharedSecrets.getJavaNioAccess();
```

Comment:
There are methods in IOUtil for the classes in sun.nio.ch. So no need to expose NIO_ACCESS here.